vagrant-libvirt.gemspec

```diff
@@ -20,9 +20,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec-expectations", "~> 2.14.0"
   gem.add_development_dependency "rspec-mocks", "~> 2.14.0"
 
-  gem.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
-  gem.add_runtime_dependency 'nokogiri', '~> 1.6.0'
   gem.add_runtime_dependency 'fog-core', '~> 1.43.0'
+  gem.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
+
+  # Make sure to use the same version of Nokogiri as Vagrant (https://git.io/v9sSO).
+  gem.add_runtime_dependency 'nokogiri', '~> 1.7.1'
```

Comment:
There was a conflict here so I pulled the Nokogiri line down, and sorted the lines above in the hope that a bit of determinism will make future changes less likely to conflict.

Reply:
@agriffis Your change works for me. Have a look at issue #828, I left a comment. Used vagrant 2.0.1 on Ubuntu 16.04 LTS. The only change needed seems to be
````
kzorba@enigma-home ~/tmp/vagrant-libvirt (master)-> git diff vagrant-libvirt.gemspec
diff --git a/vagrant-libvirt.gemspec b/vagrant-libvirt.gemspec
index f8fe7b5..a0b02b8 100644
--- a/vagrant-libvirt.gemspec
+++ b/vagrant-libvirt.gemspec
@@ -21,7 +21,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec-mocks", "~> 3.5.0"
 
   gem.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
-  gem.add_runtime_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_runtime_dependency 'nokogiri', '~> 1.6'
   gem.add_runtime_dependency 'fog-core', '~> 1.43.0'
 
   gem.add_development_dependency 'rake'
````

I think @jcf PR should be modified slightly as mentioned. I must say however I am very new to vagrant and totally ignorant about Ruby ;-)